CommandLine/CommandLine.swift

```diff
@@ -79,7 +79,16 @@ public class CommandLine {
 
     return usedFlags
   }
-
+    
+  /**
+   * If true, when the stopper is encountered '--', we not only stop parsing options in general
+   * we also stop parsing multi-value options. This allows a user to indicate to an unbounded option
+   * to stop if it happens to be the last option in the option list, just before the non-option
+   * unparsed arguments.
+   *
+   */
+  public var stopperStopsMultiValueOptionParsing = false
```

Comment:
Yeah, looking at the original project's documentation, I think they intend `--` to stop *all* command processing. If we want to merge this back into their repo (which I think would be a good thing), then I'd say we should just always make `--` stop it. No option needed.